pkg/credentialprovider/plugin/examples/acr/azure.go

```diff
@@ -0,0 +1,175 @@
+// +build !providerless
```

Comment:
This file was mostly copied from https://github.com/kubernetes/kubernetes/blob/master/pkg/credentialprovider/azure/azure_credentials.go

Reply:
thanks for adding this. I think it is OK to copy them from current acr implementations and evolve more later.